src/sentry/db/models/outboxes.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Collection, List, Mapping, Type
+
+from sentry.types.region import find_regions_for_orgs
+
+if TYPE_CHECKING:
+    from sentry.models.outbox import ControlOutboxBase, OutboxCategory, RegionOutboxBase
+
+
+class ProducesRegionOutboxOnUpdate:
+    __default_flush__ = True
```

Comment:
Won't flushing by default trade-off availability for consistency? If our auth model updates are applied eventually we should be ok.

Reply:
That's totally fair.  I will override this for the auth models.